ui/src/views/network/CreateSharedNetworkForm.vue

```diff
@@ -591,6 +633,30 @@ export default {
     isValidTextValueForKey (obj, key) {
       return this.isValidValueForKey(obj, key) && String(obj[key]).length > 0
     },
+    allowSettingMTU () {
+      api('listConfigurations', {
+        name: 'allow.end.users.to.specify.vm.mtu',
+        zoneid: this.selectedZone.id
+      }).then(json => {
+        this.setMTU = json?.listconfigurationsresponse?.configuration[0]?.value === 'true'
```

Comment:
cc this seems to be an issue in the UI; if listConfigurations can't be called by normal accounts, we need this to be passed/discoverd in the UI in some other way (a new API or extend the listCapabilities API). cc @NuxRo